File: domain/src/main/java/com/centram/common/vo/NotificationVO.java
package com.centram.common.vo;


import com.centram.domain.enumarator.NotificationType;
import com.centram.domain.enumarator.Status;
import lombok.*;

import java.io.Serializable;
import java.math.BigInteger;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
public class NotificationVO implements Serializable {
    private static final long serialVersionUID = -6523446568157662441L;
    private BigInteger id;
    private String title;
    private String body;
    private Status status;
    private NotificationType notificationType;
    private BigInteger userId;
}

File: ui/src/app/model/NotificationVO.ts
import { NotificationType } from "./enumerator/NotificationType";
import { Status } from "./enumerator/Status";

export class NotificationVO {
    id: number;
    title: string;
    body: string;
    notificationType: NotificationType;
    status: any;
    userId: number;
    constructor() {
        this.id = null;
        this.title = '';
        this.body = '';
        this.userId = null;
        this.notificationType = NotificationType.INFO;
        this.status = Status.PUSHED;
    }
}